pom.xml

```diff
@@ -1,119 +1,84 @@
-                <executions>
-                    <execution>
-                        <id>get-the-git-infos</id>
-                        <goals>
-                            <goal>revision</goal>
-                        </goals>
-                        <phase>validate</phase>
-                    </execution>
-                </executions>
-            </plugin>
-        </plugins>
-    </build>
-    <dependencies>
-        <!--Bukkit API-->
-        <dependency>
-                <groupId>org.bukkit</groupId>
-                <artifactId>bukkit</artifactId>
-                <version>1.14.4-R0.1-SNAPSHOT</version>
-                <scope>provided</scope>
-        </dependency>
-         <!--Worldguard dependency-->
-        <dependency>
-                <groupId>com.sk89q.worldguard</groupId>
-                <artifactId>worldguard-legacy</artifactId>
-                <version>7.0.0-SNAPSHOT</version>
-                <scope>provided</scope>
-                <exclusions>
-                    <exclusion>
-                        <groupId>org.bukkit</groupId>
-                        <artifactId>bukkit</artifactId>
-                    </exclusion>
-                    <!--<exclusion>-->
-                        <!--<groupId>com.sk89q.worldedit</groupId>-->
-                        <!--<artifactId>worldedit-bukkit</artifactId>-->
-                    <!--</exclusion>-->
-                    <exclusion>
-                        <groupId>com.sk89q</groupId>
-                        <artifactId>commandbook</artifactId>
-                    </exclusion>
-                    <exclusion>
-                        <groupId>org.bstats</groupId>
-                        <artifactId>bstats-bukkit</artifactId>
-                    </exclusion>
-            </exclusions>
-        </dependency>
-         <!--Vault dependency-->
-        <dependency>
-                <groupId>net.milkbowl.vault</groupId>
-                <artifactId>VaultAPI</artifactId>
-                <version>1.7</version>
-                <scope>provided</scope>
-                <exclusions>
-                    <exclusion>
-                        <groupId>org.bukkit</groupId>
-                        <artifactId>bukkit</artifactId>
-                    </exclusion>
-                </exclusions>
-        </dependency>
-    </dependencies>
+	<properties>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
 
-</project>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+	</properties>
+
+	<repositories>
+		<repository>
+			<id>spigot-repo</id>
+			<url>https://hub.spigotmc.org/nexus/content/repositories/snapshots/</url>
+		</repository>
+		<repository>
+			<id>sk89q-repo</id>
+			<url>http://maven.sk89q.com/repo/</url>
+		</repository>
+		<repository>
+			<id>jitpack.io</id>
+			<url>https://jitpack.io</url>
+		</repository>
+	</repositories>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.spigotmc</groupId>
```

Comment:
GP uses CB as the dependency, not spigot. (Yes, there are a fair bit of servers on CB.)

Reply:
I already changed it on my local branch